.env-dist

```diff
@@ -19,7 +19,8 @@ GIT_USER_EMAIL=bar@foo.com
 CRON_MAIL_TO=bar@foo.com,foo@bar.com
 YANGCATALOG_CONFIG_PATH=/etc/yangcatalog/yangcatalog.conf
 CONFD_VERSION=8.0
-CONFD_PASSWORD=<CONFD_PASSWORD>
+CONFD_PASSWORD=admin
+CONFD_ADMIN_PASSWORD=$0$admin
```

Comment:
I'd prefer something like `CONFD_PASSWORD_HASH`. `CONFD_ADMIN_PASSWORD` doesn't explain how it's different from `CONFD_PASSWORD`.

Reply:
That's true - I updated it to `CONFD_PASSWORD_HASH`.